src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -155,6 +155,25 @@ bool Transformations::fuse_type_to_convert(const std::shared_ptr<ngraph::Node>&
     return false;
 }
 
+template <typename T>
+bool Transformations::fuse_type_to_binary_comparision(const std::shared_ptr<ngraph::Node>& node,
```

Comment:
Looks like, it should be moved into the [convert_precision.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/tests/utils/convert_precision.cpp).
@itikhono, could you please take a look?

Reply:
At early, we thought only do change in cpu plugin, so for now we should do in for common transformation?